css/site.css

```diff
@@ -1066,4 +1064,48 @@ a.nav-link {
 
 .radio input[type=radio] {
   margin-top: 5px;
+}
+
+.navbar-brand:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+
+.btn:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+a.sr-only {
```

Comment:
@ostafinskim the CSS on lines 1078-1086 looks to me like it's duplicate and unneeded. These styles are already defined for `.sr-only` elements in the material.min.css file here:

https://github.com/FitzwilliamMuseum/tessitura-fitz/blob/master/css/material.min.css#L10933

Reply:
css selector changed to .doc-skip